components/chart/types/rangecolumn.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Range Column
+page_title: Chart - Range Column
+description: Overview of the Range Column Chart for Blazor.
```

Comment:
Try to add descriptions containing around 150 symbols and describing the content of the article.